doc/recognition.xml

```diff
@@ -657,7 +697,13 @@ A successful find homomorphism method has the following duties:
     group in the attribute <Ref Attr="Homom"/>. Note that if your homomorphism
     changes the representation (for example going from matrix groups to
     permutation groups), you will have to set the attribute
-    <Ref Attr="methodsforfactor"/> accordingly.
+    <Ref Attr="methodsforfactor"/> accordingly. Also,
+    <Ref Attr="ValidateHomomInput"/> may be set to a function which returns
+    <C>false</C> for values which may cause <Ref Attr="Homom"/> to
+    produce the wrong answer, or error. For example, if this node represents
+    a matrix group that preserves a subspace, then matrices which do not
+    preserve that subspace may cause <Ref Attr="Homom"/> to produce
+    incorrect answers.
```

Comment:
Is it really necessary / useful to give yet another example here? Shouldn't readers of this be expected to look at the documentation for the respective attribute for further details?